package.json

```diff
@@ -37,41 +37,44 @@
   },
   "dependencies": {
     "@types/compression": "^1.7.2",
-    "@types/express": "^4.17.13",
     "@types/express-fileupload": "^1.2.2",
     "@types/http-errors": "^2.0.0",
     "@types/node": "*",
     "compression": "1.7.4",
-    "express": "4.18.2",
     "express-fileupload": "1.4.0",
     "http-errors": "2.0.0",
     "mime": "3.0.0",
     "openapi3-ts": "3.2.0",
     "ramda": "0.28.0",
     "triple-beam": "1.3.0",
-    "typescript": "4.9.5",
```

Comment:
⚠️ Breaking: typescript becomes an optional peer, needed for the client generating only